libs/application/ui-components/src/components/NationalIdWithName/NationalIdWithName.tsx

```diff
@@ -81,13 +81,13 @@ export const NationalIdWithName: FC<
       { minAge: minAgePerson },
     )
   } else if (!errorMessage) {
-    nationalIdFieldErrors = getErrorViaPath(errors, nationaIdField)
+    nationalIdFieldErrors = getErrorViaPath(errors, nationalIdField)
   }
 
   // get default values
   const defaultNationalId = nationalIdDefaultValue
     ? nationalIdDefaultValue
-    : getValueViaPath(application.answers, nationaIdField, '')
+    : getValueViaPath(application.answers, nationalIdField, '')
```

Comment:
```suggestion
    : getValueViaPath(application.answers, nationalIdField)
```
Last input is optional and can be skipped